File: javascript/natural.js
print("Enter a number upto which you want to print the natural numbers:");
var Count = parseInt(readln());
print("The natural numbers upto ");
var NaturalNumbers = Count;
print(" are ");
var Counter = 1;
while(Counter <= Count)
{
  NaturalNumbers = NaturalNumbers + " " + Counter;
  Counter = Counter + 1;
}
print (NaturalNumbers + ".");

File: Java/natural.java
import java.util.Scanner;
class Main {
  public static void main(String[] args) {
    int Count, Counter = 1;
    Scanner s = new Scanner(System.in);
    System.out.print("Enter a number upto which you want to print the natural numbers:");
    Count = s.nextInt();
    System.out.print("The natural numbers upto " + Count + " are ");
    while(Counter <= Count)
    {
      System.out.print(" " + Counter);
      Counter = Counter + 1;
    }
    System.out.print(".");
    
  }
  }